users/templates/users/friends.html

```diff
@@ -4,23 +4,42 @@
 {% block stylesheets %}<link rel="stylesheet" type="text/css" href="{% static 'users/friends.css'%}">{% endblock %}
 {% block content %}
 
+
 <div class="friends-body">
-    <div class="followers">
-        <h1>Followers</h1>
-        <ul>
-        {% for f in followers %}
-            <li><a href={% url 'profile' f.pk %}>{{f.user.username}}</a></li>
-        {% endfor %}
-        </ul>
+    <div class="navigation">
+        <button onclick="window.location.href = '{% url 'followers' author.pk %}';">Followers</button>
+        <button onclick="window.location.href = '{% url 'following' author.pk %}';">Following</button>
+        <button class="active" onclick="window.location.href = '{% url 'friends' author.pk %}';">Friends</button>
     </div>
 
-    <div class="following">
-        <h1>Following</h1>
-        <ul>
-        {% for f in following %}
-            <li><a href={% url 'profile' f.pk %}>{{f.user.username}}</a></li>
-        {% endfor %}
-        </ul>
+    <div class="content">
+        <div class="friends">            
+            <ul>
+            {% if author and author.user == user and freqs|length  > 0 %}
+                <li>Friend Requests</li>
```

Comment:
put friend requests in a separate list, currently it looks weird having '--' as the sperator, and titles as list elements